### DISCUSSION TITLE
west flash fails with OpenOCD

### DISCUSSION THREAD

**Discussion by glenn-andrews** [zephyrproject-rtos/zephyr#33008] [Q&A] [answered]
Hi there. Apologies in advance for the beginner's question, but I've been banging my head against a brick wall for the past 2 hours.

It appears to be the same problem as seen [here](https://github.com/zephyrproject-rtos/zephyr/issues/29722) in that west is not seeing the path to OpenOCD. I'm on Windows 10 Enterprise.

I've followed the directions [here](https://docs.zephyrproject.org/latest/guides/flash_debug/host-tools.html#openocd-debug-host-tools) trying it both in the system path and the user path. I even changed the name of the environment variables from "Path" to "PATH", but to no avail.

If I've posted in the wrong place, please let me know a better venue for the question. I couldn't see many active forums for this.

**Comment by danbujak** [comment]
I'm having the same issue- did you ever sort this out?

**Comment by glenn-andrews** [comment, marked as answer]
Hi,

Yes, it was because I'd not installed OpenOCD before installing west and
running a build.

Running clean and rebuilding fixed it.

Glenn

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
